syft/pkg/cataloger/binary/cataloger.go

```diff
@@ -0,0 +1,78 @@
+package binary
+
+import (
+	"github.com/anchore/syft/syft/artifact"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/source"
+)
+
+const catalogerName = "binary-cataloger"
+
+func NewBinaryCataloger() pkg.Cataloger {
```

Comment:
nit: since the package name is already `binary`

```suggestion
func NewCataloger() pkg.Cataloger {
```